.gitignore

```diff
@@ -39,3 +39,10 @@ pom.xml.versionsBackup
 *.flattened-pom.xml
 /.apt_generated/
 /.apt_generated_tests/
+
+# Extension
+/extension/node_modules/
+/extension/dist/
+/extension/coverage/
+/extension/src/extension-config.ts
```

Comment:
This gets generated by the NPM build, right?

Reply:
Yes, exactly